ui/src/app/components/apps-panel/apps-panel-button.tsx

```diff
@@ -64,24 +64,23 @@ export const AppsPanelButton = (props: Props) => {
     <TooltipTrigger content={disabledTooltip} disabled={!showTooltip}>
       <Clickable
         {...{ disabled, onClick }}
-        style={{ padding: '0.5em' }}
+        style={{
+          ...(disabled ? buttonStyles.disabledButton : buttonStyles.button),
+          margin: '0.5em',
+        }}
+        as={FlexColumn}
```

Comment:
Takes advantage of change in buttons.tsx that allows components to be passed in the as attribute of Clickable.